crates/shimkit/Cargo.toml

```diff
@@ -0,0 +1,109 @@
+[package]
+name = "shimkit"
+description = "Opinionated library for building containerd shims"
+version = "0.3.1"
```

Comment:
question: why does shimkit starts with version 0.3.1?

Reply:
I had published shimkit as an experiment a long time ago, and was on 0.2. If we use containerd-shimkit we can start from 0.1.
The only caveat is that we need to publish it for CI to pass!

Reply:
> The only caveat is that we need to publish it for CI to pass!

why is that?

Reply:
Otherwise the step that check if containerd-shim-wasm can be published fails, because it uses an unpublished dependency